smartsim/_core/entrypoints/colocated.py

```diff
@@ -165,21 +165,27 @@ def main(
 ):
     global DBPID
 
+    lo_address = current_ip("lo")
     try:
-        ip_address = None
-        if network_interface:
-            ip_address = current_ip(network_interface)
-        lo_address = current_ip("lo")
+        ip_addresses = []
+        ip_addresses.extend(
+            [current_ip(interface) for interface in network_interface.split(",")]
+        )
```

Comment:
Can these lines be simplified to just be
```py
ip_addresses = [current_ip(interface) for interface in network_interface.split(",")]
```

Reply:
Yes, I think it makes sense.